src-tauri/tauri.conf.json

```diff
@@ -87,6 +87,11 @@
             "cmd": "impacket-secretsdump",
             "args": true
           },
+          {
+            "name": "sudo",
```

Comment:
I feel a bit sketchy about making sudo an accepted command. And it _usually_ needs credentials which makes life even more difficult.

Reply:
Perhaps we can make assumptions that the application is run as root?